core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -220,7 +219,7 @@ protected void tryComplete() {
 
     synchronized (this) {
       if (state == STATE_RESPONSE && allChildrenComplete) {
-        if (LOGGER.isDebugEnabled()) {
+        if (debugLogEnabled) {
           LOGGER.debug(this + " completed.");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use the built-in formatting to construct this argument. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLOTwXotIXnmsUkh18&open=AWkLOTwXotIXnmsUkh18)

[ISSUE_KEY]: AWkLOTwXotIXnmsUkh18